techmatters/flex-plugins
# CHI-1735: fix clear form, mapping tweaks

Primary reviewer:

## Description

* Fixes 'clear form' for resources search
* Adds full descriptions of all filters being used on results page
* Fixes some mappings and makes conversion mapping code more strongly typed
* Adds full list of Canadian Cities as provided by KHP

We will do full translations once we are getting ready to launch & UIs are finalised

### Checklist
- N/A Feature flags added
- N/A Strings are localized

### Related Issues
Fixes CHI-1735

### Verification steps

* Deploy to Aselo Development
* Ensure cities appear when provinces are selected (we don't seem to have any for Quebec, checking that)
* Try various selections and ensure the descriptions on the results page reflect the selections
* Experiment with filters and see if they return sensible results
